0_intro/exercises/broken_ownership/src/lib.rs

```diff
@@ -18,13 +18,13 @@ fn modify_values(values: Vec<i32>) {
 // FIXME
 pub fn print_hello() {
     let mut msg = String::from("hello world");
-    let word = first_word(&msg); // Get a reference to the first word
+    let word = first_word(&mut msg); // Get a reference to the first word
 
     msg.clear(); // Empty the message
     println!("The first word is: {}", word); // Print first word
 }
 
-fn first_word(msg: &String) -> &str {
+fn first_word(msg: &str) -> &str {
```

Comment:
This exercise doesn't work yet. When I try and compile it, I get an error about line 23. You fixed a lot of the issues already tho, so that's really good. Try to look at the order in which the functions are called